react/the_react_ecosystem/styling_react_applications.md

```diff
@@ -0,0 +1,66 @@
+### Introduction
+
+Up until now you've been using vanilla CSS to style your React projects, but there are other options available. This lesson is intended to be more like a guide or a list of options for you to explore.  
+
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- Know about ways to style React applications 
+
+### Vanilla CSS 
+
+Vanilla CSS is the simplest way to style. In the previous courses, you've learned a lot of CSS and all of those skills are applicable to React. There are a couple of things we'd like to highlight.
+
+1. CSS Modules
+
+    CSS Modules let you write CSS style declarations that are scoped locally.
+
+1. CSS Utility Frameworks
+
+    CSS frameworks are popular choice of styling React applications. They provide a set of pre-defined classes that you can directly use in your HTML, or JSX in our case. [Tailwind CSS](https://tailwindcss.com) is by far the most popular choice. 
+
+
+### CSS in JS
+
+Why even write CSS in CSS when you can write it in JavaScript? Just kidding, of course! 
+
+CSS-in-JS is a paradigm for styling front-end projects. It allows you to take entire control of CSS with JavaScript and extends it with various features. Additionally, it also helps to apply styling in a logical fashion i.e. based on state. There are various CSS-in-JS solutions. One of the most popular ones in the React ecosystem is [styled-components](https://styled-components.com/).
+
+### Component Libraries
+
+What if everything's already done for you? Styling, behavior, and accessibility are taken care for you in component libraries. As the name suggests, these libraries provide adaptable and reusable components that you can use directly in your project. These components include, but are not limited to, dropdowns, drawers, calendars, toggles, tabs, and all other components you can think of. 
+
+[Material UI](https://mui.com/), [Radix](https://www.radix-ui.com/), and [Chakra UI](https://chakra-ui.com/) are worth a mention when talking about component libraries.
+
+<div class="lesson-note lesson-note--tip" markdown="1" >
+
+For the purposes of learning and this course, we advise you to implement your component's styling from scratch i.e. use vanilla CSS or a CSS-in-JS option. 
```

Comment:
```suggestion
For learning purposes throughout this course, we advise you to implement your component's styling from scratch i.e. use vanilla CSS or a CSS-in-JS option. 
```

`For the purposes of learning and this course` doesn't sound entirely right to me. Maybe you have another way to put it?